overview/prebid-universal-creative.md

```diff
@@ -11,40 +11,99 @@ nav_section: intro
 # Prebid Universal Creative
 {:.no_toc}
 
-The Prebid Universal Creative makes it easier for publishers to configure Prebid in their ad server. The Prebid Universal Creative provides a single creative configuration that can be used across many formats, platforms, devices, and ad servers.
+* TOC
+{:toc}
 
-Specifically, you need to use the Universal Creative in these scenarios:
+## Overview
 
-- AMP and Prebid SDK (these require loading creatives from cache)
-- when you need to support safeframes
-- when you need to support native
+The Prebid Universal Creative (PUC) is a collection of rendering routines
+that can pull a particular ad ID from Prebid's cache and do the right
+thing to display it. The scripts are generally entered into the ad server for
+when a Prebid ad has won the auction. There are a number of use cases:
+
+{: .table .table-bordered .table-striped }
+| Use Case | PUC file | Alternate Approach |
+| --- | --- | --- |
+| web banner: iframe | creative.js | [Banner and Outstream Video iframes](#banner-and-outstream-video-iframes) |
+| web banner: safeframe | creative.js | [Banner Safeframes](#banner-safeframes) |
+| web outstream video: iframe | creative.js | [Banner and Outstream Video iframes](#banner-and-outstream-video-iframes) |
+| web outstream video: safeframe | n/a | n/a |
```

Comment:
should we note that the file works in this scenario but bypasses the frame?

Reply:
That seems unsettling, if not wrong. How does it know _where_ to write the ad again? We can't document that the scenario is supported unless we have the details. And I don't.

Reply:
Dug into how this works for one particular outstream renderer, APEX2. It assumes that adunit code is a div and draws it there.